.golangci.yml

```diff
@@ -26,6 +26,12 @@ linters:
     - depguard
     - goimports
 
+issues:
+  exclude-rules:
+    - linters:
+        - staticcheck
+      text: "SA1019"
```

Comment:
this change to disable staticcheck from disallowing using deprecated fields.

Reply:
👍